installer.sh

```diff
@@ -95,19 +94,28 @@ ensurepath() {
     IFS="$_save_ifs"
 
     case "${SHELL:-/bin/sh}" in
-      */bash) _rcfile=".bashrc" ;;
-      */zsh) _rcfile=".zshrc" ;;
-      *) _rcfile=".profile"
+    */bash) _rcfile=".bashrc" ;;
+    */zsh) _rcfile=".zshrc" ;;
+    *)
+        _rcfile=".profile"
         ;;
     esac
 
-    say "" >&2
-    say "Add $_required to your path" >&2
-    # shellcheck disable=SC2068,SC2016
-    say 'HINT:   echo '\''export PATH="$HOME/.local/bin:$PATH"'\'" >> ~/${_rcfile}" >&2
-    say "Update your current terminal." >&2
-    # shellcheck disable=SC2068,SC2016
-    say 'HINT:   export PATH="$HOME/.local/bin:$PATH"' >&2
+    echo -e "\e[1;31m\n[ATTENTION REQUIRED]\e[0m"
+    echo -e "\e[1;34mTo ensure your system can find the Golem binaries, please include '$_required' within your path, by following the instructions below.\e[0m"
+    echo -e "\e[1;33m1. Add the path to your configuration file:\e[0m"
+    echo -e "\e[0;32m   echo 'export PATH=\"\$HOME/.local/bin:\$PATH\"' >> ~/${_rcfile}\e[0m"
+    echo -e "\e[1;33m2. Apply the changes in your current terminal:\e[0m"
+    echo -e "\e[0;32m   export PATH=\"\$HOME/.local/bin:\$PATH\"\e[0m"
```

Comment:
As I understand, this will only show up if we don't have yagna binaries in path, correct?

Reply:
Yes, correct. I don't see it on my macbook when installing due to me having it in PATH already, where as on my RPi I see the warning every single time because I haven't added it to path yet.